src/Commands.pm

```diff
@@ -2603,7 +2594,7 @@ sub cmdGmrecall {
 	}
 
 	my $packet = pack("C*", 0xBC, 0x01).pack("a24", $args);
```

Comment:
if i understand well, this line now do nothing.
Shoun't be removed?

Reply:
missed that, fixed